test/integration_test.py

```diff
@@ -998,6 +998,26 @@ def assert_manifest(expected_bundles):
             # Without credentials, only the public bundle should be represented
             assert_manifest({public_bundle})
 
+            # Create a single-file curl manifest and verify that the OAuth2
+            # token is present in the .curlrc
+            managed_access_file_id = self.random.choice(
+                self.random.choice(hits)['files']
+            )['uuid']
```

Comment:
```suggestion
            managed_access_file = self.random.choice(
                self.random.choice(hits)['files']
            )
```

In general, I'm not a big fan of these continuations.

Reply:
`_test_managed_access` is getting too big. We should break it up into methods and sub tests where possible.  Please add FIXME for this.

Reply:
https://github.com/DataBiosphere/azul/issues/3893